drivers/volume/volume.go

```diff
@@ -364,6 +364,9 @@ type Driver interface {
 	//GetAutoFsTrimStatus get status of autofstrim
 	GetAutoFsTrimStatus(pxEndpoint string) (map[string]api.FilesystemTrim_FilesystemTrimStatus, error)
 
+	//GetAutoFsTrimUsage get usage stats of autofstrim
+	GetAutoFsTrimUsage(endpoint string) (map[string]*api.FstrimVolumeUsageInfo, error)
```

Comment:
Can we change the input variable name to `pxEndpoint` just like the function above for the sake of uniformity?

Reply:
fixed